### PR TITLE
[DOPS-3440] Document how to get the plugin to work on our Jenkins

### PR DESCRIPTION
**Pull request by mbtiii**
Hopefully we won't need this soon, as they've got rid of the version check in their own plugin! 🤞

### CODE REVIEW

File: CARPE_README.md [hunk 1]
```diff
@@ -12,10 +12,33 @@ Our Jenkins UI is getting locked on multi-branch pipelines after some indetermin
...
-$ cd devops-sonar-scanner-jenkins
-$ git checkout  DOPS-3438-modify-the-sonar-jenkins-plugin
-$ ./mvnw clean install -Dmaven.test.skip=true
-```
\ No newline at end of file
+VERSION="2.15.6"
+aws s3 cp target/sonar.hpi s3://carpe-jenkins/plugins/${VERSION}/sonar.hpi
+```
+
+After publishing the plugin, you will need to go to the Jenkins project and change the location in S3 that the plugin is being pulled from to match the version that you just published.
```
Comment (BenMcClainCarpe) [inline CARPE_README.md]: > you will need to go to the Jenkins project and change the location in S3 that the plugin is being pulled from

Where specifically?

**Review by BenMcClainCarpe — CHANGES_REQUESTED** [review]
Thought there couldn't be any "request changes" on this little PR did you!!!??

File: CARPE_README.md (hunk 1 above)
Comment (mbtiii) [reply CARPE_README.md]: In a file to which I have just linked!

File: CARPE_README.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ VERSION="2.15.6"
 aws s3 cp target/sonar.hpi s3://carpe-jenkins/plugins/${VERSION}/sonar.hpi
 ```
 
-After publishing the plugin, you will need to go to the Jenkins project and change the location in S3 that the plugin is being pulled from to match the version that you just published.
+After publishing the plugin, you will need to go to the Jenkins project and [change the location in S3 that the plugin is being pulled from](https://github.com/carpe/devops-jenkins/blob/master/ami-creation/jenkins-controller/files/jenkins-startup.sh) to match the version that you just published.
```
Comment (BenMcClainCarpe) [inline CARPE_README.md]: No, you mean here!  https://github.com/carpe/devops-jenkins/blob/d0d2d689daac5e671c1599ed9bfc2beb89ed46ed/ami-creation/jenkins-controller/files/configure-jenkins.sh#L50

**Review by BenMcClainCarpe — CHANGES_REQUESTED** [review]

File: CARPE_README.md (hunk 2 above)
Comment (mbtiii) [reply CARPE_README.md]: I actually mean HERE! https://github.com/carpe/devops-jenkins/blob/d0d2d689daac5e671c1599ed9bfc2beb89ed46ed/ami-creation/jenkins-controller/files/configure-jenkins.sh#L50-L51

**Review by BenMcClainCarpe — APPROVED** [review]
🤞

**Pull request merged**